pkg/queue/cluster_queue_impl.go

```diff
@@ -221,6 +222,25 @@ func (c *clusterQueueBase) DumpInadmissible() (sets.Set[string], bool) {
 	return elements, true
 }
 
+func (c *clusterQueueBase) Snapshot() ([]*workload.Info, bool) {
+	totalLen := c.heap.Len() + len(c.inadmissibleWorkloads)
+	if totalLen == 0 {
```

Comment:
I don't think we need this special case. If the queue is empty we want to return empty list, rather than nil. Nil means the status is not computed yet.

Reply:
check removed